tmt-app/src/Components/Quiz.jsx

```diff
@@ -29,6 +28,27 @@ function Quiz() {
           </Grid>
         ))}
       </Grid>
+      <Link
+        sx={{ color: 'Boba.main' }}
+        underline="none"
+        aria-label="Learn more about other different majors"
+        component={RouterLink}
+        to="/quiz/skills"
+      >
+        <PrimaryBtn
+          sx={{
+            backgroundColor: 'Matcha.main',
+            '&.MuiButtonBase-root': {
+              padding: {
+                xs: '1rem 2.5rem',
+                xl: '1rem 3.25rem',
+              },
```

Comment:
Can we not just add padding directly here? Why do we need to override with the button root class?